stable/rbac-manager/templates/deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       containers:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        imagePullPolicy: Always
```

Comment:
Nice.

Reply:
Woops, didn't actually mean to do that - that was a relic of testing chart upgrades.